### PR TITLE
Update template render when jwt auth enabled

### PR DESCRIPTION
**Pull request by tuteng**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Review by urfreespace — DISMISSED** [review]

File: charts/sn-platform/templates/proxy/proxy-cluster.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ spec:
 {{- with .Values.proxy.labels }}
 {{ toYaml . | indent 6 }}
 {{- end }}
-{{- if or (and .Values.prometheus.scrape.proxy .Values.prometheus.scrape.enabled) .Values.monitoring.datadog .Values.proxy.annotations }}
+{{- if or (and .Values.prometheus.scrape.proxy .Values.prometheus.scrape.enabled) .Values.monitoring.datadog .Values.proxy.annotations .Values.auth.oauth.enabled .Values.auth.authentication.jwt.enabled }}
```
Comment (FushuWang) [inline charts/sn-platform/templates/proxy/proxy-cluster.yaml:60]: It seems the `Values.auth.oauth.enabled` and `.Values.auth.authentication.jwt.enabled` are put to the wrong place

Comment (FushuWang) [reply charts/sn-platform/templates/proxy/proxy-cluster.yaml:60]: It should be at line 73?

Comment (tuteng) [reply charts/sn-platform/templates/proxy/proxy-cluster.yaml]: Great catch, thanks, PTAL @FushuWang

**Comment by ericsyh** [comment]
@FushuWang @maxsxu @ciiiii @urfreespace PTAL and reivew the PR.

File: charts/sn-platform/templates/proxy/proxy-cluster.yaml [hunk 2]
```diff
@@ -80,6 +80,9 @@ spec:
 {{- with .Values.proxy.extraSecretRefs }}
 {{ toYaml . | indent 4 }}
 {{- end }}
+    {{- if or .Values.auth.oauth.enabled .Values.auth.authentication.jwt.enabled }}
```
Comment (FushuWang) [inline charts/sn-platform/templates/proxy/proxy-cluster.yaml:83]: I just found there is a duplicated code at lines 104-107.

https://github.com/streamnative/charts/blob/fixed/update-template-render/charts/sn-platform/templates/proxy/proxy-cluster.yaml#L104-L107

**Review by FushuWang — APPROVED** [review]

**Pull request merged**